pkg/cap/anchor/cluster/server.go

```diff
@@ -0,0 +1,135 @@
+package cluster
+
+import (
+	"context"
+	"time"
+
+	"capnproto.org/go/capnp/v3/server"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/wetware/casm/pkg/cluster"
+	"github.com/wetware/casm/pkg/cluster/routing"
+	api "github.com/wetware/ww/internal/api/cluster"
+)
+
+var defaultPolicy = server.Policy{
+	// HACK:  raise MaxConcurrentCalls to mitigate known deadlock condition.
+	//        https://github.com/capnproto/go-capnproto2/issues/189
+	MaxConcurrentCalls: 64,
+	AnswerQueueSize:    64,
+}
+
+type ClusterServer struct {
+	node *cluster.Node
+	ctx  context.Context
+}
+
+func (rs *ClusterServer) NewClient(policy *server.Policy) ClusterClient {
+	return ClusterClient{api.Cluster_ServerToClient(rs, policy)}
+}
+
+func (rs *ClusterServer) Iter(ctx context.Context, call api.Cluster_iter) error {
+	h := serverIterator{
+		handler: call.Args().Handler().AddRef(),
+		bufSize: call.Args().BufSize(),
+	}
+	it := rs.node.View().Iter()
+
+	go func() {
```

Comment:
Contrary to pubsub subscriptions, this is not intended to be a long-running background task, so I think we should perform the streaming synchronously on the server-side.  In addition to being "more correct" from a capnp RPC point-of view (it will count against resource limits & flow-control), this will also facilitate reference management, and therefore simplify the implementation of `serverIterator`.

However, the streaming is expected to happen on the order of ms to seconds, so we should definitely call `call.Ack()` before streaming.